src/mSupplyMobileApp.js

```diff
@@ -151,10 +150,10 @@ class MSupplyMobileAppContainer extends React.Component {
   };
 
   synchronise = async () => {
-    const { syncState, dispatch } = this.props;
+    const { dispatch, isSyncing } = this.props;
     const { isInitialised } = this.state;
 
-    if (!isInitialised || syncState.isSyncing) return; // Ignore if syncing.
+    if (!isInitialised || isSyncing) return;
```

Comment:
Not having this component listen for `sync` changes - they happen quite frequently - especially after syncing has begun (for every increment of the progress), so just listening for when syncing starts/stops